app/models/metric/targets.rb

```diff
@@ -32,20 +32,15 @@ def self.capture_cloud_targets(zone, options = {})
     # 1) need to find all the VMs under all the availability zones
     # 2) need to find all the VMs that may not be in an availability zone
     # 3) cloudy clusters?
```

Comment:
At the very least, these comments should reflect that we're no longer going from the availability zone down.
We should also mention having an availabilty zone with perf_capture_enabled,  "under management (has ems)" and "running" (state on), anything else to clear up the changed logic?